utilities/common.py

```diff
@@ -127,14 +128,27 @@ def get_binning_fakes_pt(min_pt, max_pt):
     #edges = np.append(edges, [max_pt])
     return edges
 
-def get_binning_fakes_mt(mt_cut=40):
+
+def get_binning_fakes_mt(mt_cut=40, high_mt_bins=False):
     edges = np.array([0, int(mt_cut/2.), mt_cut])
```

Comment:
What's the reason to even have the bin boundary 20 GeV here?

Reply:
I think it's to have the 2 regions at low mt, 0-20 and 20-40

Reply:
Right